From 78539e323954b6967444ba39cda1125e9b7c0b85 Mon Sep 17 00:00:00 2001
From: kxxt <rsworktech@outlook.com>
Date: Thu, 23 Jan 2025 10:38:02 +0800
Subject: [PATCH] Neutralize sepolicy freeze test

We are essentially a fork of Android changing the public sepolicy

Change-Id: Ie3ddaf051729c82a0d89122d5997ed7be6ff095d
---
 tests/sepolicy_freeze_test.py  | 3 ++-
 tests/treble_sepolicy_tests.py | 1 +
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/sepolicy_freeze_test.py b/tests/sepolicy_freeze_test.py
index fa05eb193..eaa7d448e 100644
--- a/tests/sepolicy_freeze_test.py
+++ b/tests/sepolicy_freeze_test.py
@@ -61,7 +61,7 @@ def do_main():
         results += "The following public attributes were added:\n" + ", ".join(added_attributes) + "\n"
 
     if results:
-        sys.exit(f'''{results}
+        print(f'''{results}
 ******************************
 You have tried to change system/sepolicy/public after vendor API freeze.
 To make these errors go away, you can guard types and attributes listed above,
@@ -71,6 +71,7 @@ See an example of how to guard them:
     https://android-review.googlesource.com/3050544
 ******************************
 ''')
+        sys.exit(0)
 
 if __name__ == '__main__':
     do_main()
diff --git a/tests/treble_sepolicy_tests.py b/tests/treble_sepolicy_tests.py
index ff1a34817..c1b9ac350 100644
--- a/tests/treble_sepolicy_tests.py
+++ b/tests/treble_sepolicy_tests.py
@@ -51,6 +51,7 @@ def TestNoUnmappedNewTypes(base_pub_policy, old_pub_policy, mapping):
         ret += "See examples of how to fix this:\n"
         ret += "https://android-review.googlesource.com/c/platform/system/sepolicy/+/781036\n"
         ret += "https://android-review.googlesource.com/c/platform/system/sepolicy/+/852612\n"
+        ret = ""
     return ret
 
 ###
-- 
2.39.5

